src/WebJobs.Script.WebHost/Middleware/AllowSynchronousIOMiddleware.cs

```diff
@@ -16,15 +16,15 @@ public AllowSynchronousIOMiddleware(RequestDelegate next)
             _next = next;
         }
 
-        public async Task Invoke(HttpContext context)
```

Comment:
Removed unnecessary async when I could throughout

Reply:
I think removing these unnecessary async calls should improve cold start as well :-)

---
In reply to: [413441546](https://github.com/Azure/azure-functions-host/pull/5897#discussion_r413441546) [](ancestors = 413441546)